src/client/pages/OfferNew/Checkout/UserDetailsForm.tsx

```diff
@@ -64,6 +63,14 @@ export const UserDetailsForm: React.FC<Props> = ({
     )
   }
 
+  const handleSsnBlur = () => {
```

Comment:
Is this `onBlur` handler really necessary? I mean the `onSsnChange` will run when `isValidSsn` is true anyway, thanks to `handleSsnChange`?

Reply:
Or am I missing something?

Reply:
Dunno really, just wanted remove it as an inline function to keep the component cleaner :) But I will use it when adding FE validation 🙌

Reply:
Yes, I'm all for cleanliness! 👌 Just realised that it probably doesn't do anything right now. But I guess we can leave it then if you'll use it later on, although I think as a principle it's better to remove stuff that's not used in order to make the PR easier to grasp. And then you can add it again when you need it 😊

Reply:
Yes, I'm all for cleanliness! 👌 Just realised that it probably doesn't do anything right now. But I guess we can leave it then if you'll use it later on, although I think as a principle it's better to remove stuff that's not used in order to make the PR easier to grasp. And then you can add it again when you need it 😊